openflow15/match.go

```diff
@@ -407,16 +411,17 @@ func DecodeMatchField(class uint16, field uint8, length uint8, hasMask bool, dat
 		case OXM_FIELD_TCP_FLAGS:
 			val = new(TcpFlagsField)
 		default:
-			log.Printf("Unhandled Field: %d in Class: %d", field, class)
+			klog.Warningf("Unhandled Field: %d in Class: %d", field, class)
 		}
 
 		if val == nil {
-			log.Printf("Bad pkt class: %v field: %v data: %v", class, field, data)
+			klog.Errorf("Bad pkt class: %v field: %v data: %v", class, field, data)
```

Comment:
This log is useless since the error is returned?

Reply:
Yes, legacy code again. Let clean it up.